libs/wingsdk/src/target-tf-aws/react-website.ts

```diff
@@ -0,0 +1,73 @@
+import { execSync } from "child_process";
+import { existsSync, unlinkSync } from "fs";
+import { join } from "path";
+import { Construct } from "constructs";
+import { Website } from "./website";
+import { core } from "..";
+
+import { S3Object } from "../.gen/providers/aws/s3-object";
+import * as cloud from "../cloud";
+import { Connections } from "../core";
+import * as ex from "../ex";
+
+/**
+ * AWS implementation of `ex.ReactWebsite`.
+ *
+ * @inflight `@winglang/sdk.cloud.IReactWebsiteClient`
+ */
+export class ReactWebsite extends ex.ReactWebsite {
```

Comment:
Feels like this should be shared across all non-sim implementations.

Reply:
since it doesn't support content type, I can add an issue (also for the cloud.website files)

Reply:
I tend to agree, but we still don't know if all of them are implementing website host using a bucket... it should be considered when developing to different target, I'll add this point to the other targets tickets

Reply:
I think `ReactWebsite` should be implemented using a `Website`, no? And as far as I recall, there's an API for [adding a JSON](https://www.winglang.io/docs/standard-library/cloud/website#@winglang/sdk.cloud.Website.addJson) config to the website.

I don't see a need for `ReactWebsite` to access the underlying bucket. It should just use the `cloud.*` abstractions.